assets/scss/7-layout/_content-grid.scss

```diff
@@ -0,0 +1,32 @@
+// Content Grid (l-content-grid)
+//
+// Standard two col grid with a content area and sidebar. We use CSS variables to allow for flexibility in setting column ratios, order, and gap size. This grid starts at `bp-l`. The default sidebar width is to account for 300x250 ads.
+//
+// Markup: 7-layout/content-grid.html
+//
+// Styleguide 7.0.1
+//
+:root {
+  --content-grid-order: 'content side';
+  --content-grid-template: 1fr 310px;
+  --content-grid-gap: #{$size-xxl};
+}
```

Comment:
does it feel weird to include css vars here? I liked having them close to the helper, but I can also see the case for having them in `/1-settings/`

Reply:
I like having them here.